tests/testthat/setup.R

```diff
@@ -148,7 +151,10 @@ create_testing_package <- function(
     #' Adds 1 to x
     #' @param x a number
     #' @export
-    myfunction <- function(x) { x + 1}
+    myfunction <- function(x) {
+      checkmate::assert_numeric(x)
+      x + 1
+    }
     ", .open = "{{", .close = "}}")
```

Comment:
I like these changes. `add_imports` is more explicit in what it's doing, and it's nice that the template function now uses an import by default (and a package we already have installed)